pkg/querier/vcs/token.go

```diff
@@ -102,14 +107,14 @@ func tokenFromRequest(ctx context.Context, req connect.AnyRequest) (*oauth2.Toke
 	return token, nil
 }
 
-// encodeToken encrypts then base64 encodes an OAuth token.
-func encodeToken(token *oauth2.Token, key []byte) (*http.Cookie, error) {
+// Deprecated: encodeTokenInCookie creates a cookie by encrypting then base64 encoding an OAuth token.
```

Comment:
perhaps we should explain why this is deprecated and if we have any plans on removing this soon

Reply:
Yeah, agree. In the past I've seen people leave target dates/releases of when something can get removed. I.e.
> Remove this after the v1.11 release

I've found this to be useful because it helps signal to other readers (or myself later) that this bit of code is living on borrowed time.

Reply:
done by linking the frontend issue that does that. It's difficult to estimate a  release version where this is not needed and can be deleted, as I have no idea on the priority of the frontend part to be completed